src/services/nativeAuth/helpers/getLatestBlockHash.ts

```diff
@@ -38,7 +47,8 @@ const getLatestBlockHashFromServer = retryMultipleTimes(
 
 export async function getLatestBlockHash(
```

Comment:
add 
// TODO: make getLatestBlockHash accept an object and make `getBlockHash` param mandatory (to replace axios call)